content/en/community/_index.md

```diff
@@ -43,11 +42,12 @@ menu:
 {{< page/toc collapsed=true placement="inline" >}}
 </div>
 
+
 <div class="row">
 <div class="col-12 col-lg-8">
 
 {{% alert color="success" %}}
-  <i class='fas fa-users mr-1'></i> Our community values respect and
+  `<i class='fas fa-users mr-1'></i>` Our community values respect and
```

Comment:
It looks like this update is not rendering correctly, the deploy preview here shows html tags https://deploy-preview-918--etcd.netlify.app/community/

![image](https://github.com/user-attachments/assets/012104cf-a8dc-456f-9e72-3927b1c94f55)

Reply:
Ok. I'm getting it done right away.